agent/src/agent/monitoring/__init__.py

```diff
@@ -0,0 +1,30 @@
+import anodot
+
+from . import metrics, streamsets
+from agent.modules import constants, logger
+from datetime import datetime
+from prometheus_client import generate_latest
+
+logger_ = logger.get_logger(__name__)
+
+
+def get_latest():
+    streamsets.get_metrics()
```

Comment:
could we give different names to functions like get_metrics()? It's very confusing to see a getter that doesn't return anything

Reply:
pull_metrics is good enough?

Reply:
yep